service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AisConsentCreationSteps.java

```diff
@@ -323,6 +328,26 @@ public void receiveAccountData() {
 
     assertThat(actualResponse.getBody().getAccounts().size(),
         equalTo(context.getConsentAccountAccess().getAccounts().size()));
+
+    if (!context.isWithBalance()) {
+      actualResponse.getBody().getAccounts()
+          .forEach(accountDetails -> assertNull(accountDetails.getBalances()));
+    } else {
+      List<AccountReference> balances = context.getConsentAccountAccess().getBalances();
+
+      if(balances == null){
```

Comment:
Formatting (missing space)

Reply:
Done. But why was our code formatting not automatically checked?